react/__mocks__/mockSummaries.tsx

```diff
@@ -0,0 +1,64 @@
+import { Address } from 'vtex.checkout-graphql'
+import { CountryDescription } from '../mocks'
```

Comment:
You could import the actual values from mocks, so you wouldn't need to re-write them here:
```suggestion
import { countryDescriptions } from '../mocks/CountryDescriptions'
```

Reply:
Great, I'll do it this way, I was planning on permanently deleting the `mocks/` folder to only have `__mocks__`, since it's kind of confusing now, but since I'm not deleting it now to display that dropdown component, this modification will make the code cleaner.